node/README.md

```diff
@@ -147,7 +151,17 @@ $ docker run -ti coop.rchain/rnode
 (...)
 ```
 
-##### 2.1.2. via Java
+To use both the peer-to-peer and REPL capabilities of RNode, two containers running RNode need to be connected to one user-defined network bridge:
+
+```bash
+> docker network create rnode-net
+
+> docker run -dit --name rnode0 --network rnode-net coop.rchain/rnode:latest -s
+
+> docker run -it --name rnode-repl --network rnode-net coop.rchain/rnode:latest --grpc-host rnode0 -r
```

Comment:
Is this supposed to be coop.rchain ?

Reply:
ach, looks like some changes got clobbered